packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx

```diff
@@ -52,6 +70,45 @@ export const EventTypeWalletBalanceRow: React.FC<
     didFetch.current = true;
   }, [alertName, alerts]);
 
+  const subscribeAlert = useCallback(
+    async (
+      alertDetail: Readonly<{
+        eventType: EventTypeItem;
+        inputs: Record<string, unknown>;
```

Comment:
is this always `{}`?

Reply:
oh.. yeah, this `inputs` field is always empty object. The reason why it exists is for the interface consistency of [subscribeAlertByFrontendClient](https://github.com/notifi-network/notifi-sdk-ts/blob/b6881792f87ffecb8713ce83aa4a23eaecb2853f/packages/notifi-react-card/lib/utils/frontendClient.ts#L10). Hope it make sense.